examples/cis3-nft-sponsored-txs/src/lib.rs

```diff
@@ -588,81 +561,18 @@ fn contract_view<S: HasStateApi>(
 
     let all_tokens = state.all_tokens.iter().map(|x| *x).collect();
 
-    let mut all_public_keys_and_nonces = Vec::new();
-    for (account_address, public_key) in state.public_key_registry.iter() {
-        all_public_keys_and_nonces.push((*account_address, *public_key));
+    let mut all_nonces = Vec::new();
+    for (account_address, nonce) in state.nonces_registry.iter() {
+        all_nonces.push((*account_address, *nonce));
     }
```

Comment:
I think you could do this:
```suggestion
    let all_nonces: Vec<_> = state.nonces_registry.iter().copied().collect();
```